eks-anywhere-common/Addons/Core/Daily-S3-Exporter/manifestCronJob.yaml

```diff
@@ -34,12 +34,14 @@ spec:
             - -c
             - >-
                 kubectl version &&
-                kubectl get all -A > exporter.txt &&
-                cat exporter.txt &&
+                kubectl get all -A > exporter.txt && 
+                EKS_DEPLOYMENT_MODEL=$(kubectl get cluster -o json | jq -r '.items[].spec.controlPlaneConfiguration.machineGroupRef.kind') &&
+                kubectl get ns | grep -v 'default\|capas-system\|capi-kubeadm-bootstrap-system\|capi-kubeadm-control-plane-system\|capi-system\|cert-manager\|eksa-packages\|eksa-packages-elas-cluster\|eksa-system\|etcdadm-bootstrap-provider-system\|etcdadm-controller-system\|external-secrets\|flux-system\|kube-node-lease\|kube-public\|kube-system\|var-expansion\|sonobuoy\|NAME\|botkube' | awk '{print $1;}' > namespaces.txt &&
```

Comment:
This approach is not reliable as we may be installing something in the future that is not part of this list or EKS-A adds a component that will screw up the list. 
I propose we annotate or label each partner namespace with a label and use that. We only have a handful now.
The change README and call out that namespace must be labeled, so that partners do it right off the bat.

Reply:
This is done, please check.